CONTRIBUTING.md

```diff
@@ -0,0 +1,128 @@
+
+### Database identifiers (``database-name/database-version/database-identifiers.tsv``; optional)
+Contributors may provide database identifiers associated with each member of the mock community (useful, for example, for identifying associated sequences in the reference database). Each taxonomic name listed in the ``taxonomy.tsv`` in the same directory must be included, and can have zero or more database identifiers associated with it. The taxonomic name and all database identifiers should be separated by tabs.
+
+## Classic BIOM-formatted tables
+
+Several files are described as being in classic biom-format tables. In these files, the first line must begin with the text ``#Taxonomy``, followed by a tab-separated list of one or more sample identifiers. All sample identifiers provided here must be present in ``sample-metadata.tsv``. Each subsequent line should begin with the taxonomic name, followed by a tab-separated list of the relative abundances in each sample. The relative abundances must sum to 1.000 (to three decimal places) for each sample.
+
+## Submitting to mockrobiota
+mockrobiota is hosted on [GitHub](http://www.github.com), and we use GitHub's [Pull Request](https://help.github.com/articles/using-pull-requests) mechanism for reviewing and accepting submissions. To contribute to mockrobiota, we recommend following [scikit-bio's instructions for submitting a contribution](https://github.com/biocore/scikit-bio/blob/master/CONTRIBUTING.md#submitting-code-to-scikit-bio).
+
+## Compiling expected taxonomy files
+
+Expected composition data will consist of one of the following types:
+* Microbiome mock community: expected taxonomic composition for a mixture of microbial cells. The taxonomic annotations present in the expected data will be specific to the database version that is used for analysis, and will be meaningless if used for different database versions. Likewise, they may not match the source annotation (i.e., the taxonomy of each strain to the best knowledge of the mock community’s creator) if taxonomic annotations have been revised or if the reference database being used does not contain a given taxonomy.
+* Metagenome mock community: expected gene composition for a mixture of microbial cells/genomes. Gene annotations will be reference database specific, as for microbiome mock communities above.
+
+Other mock community data types are theoretically possible, and could be included in mockrobiota, which only defines required information, files, and file formats. Expected data definitions can expand as other mock community data types are contributed to mockrobiota. If submitting a mock community that is neither marker-gene sequences nor shotgun metagnome sequences, submit an [issue](https://github.com/caporaso-lab/mockrobiota/issues) to confer with the mockrobiota developers before proceeding.
+
+Here we illustrate how expected taxonomy might be compiled from source taxonomy for an example dataset.
+
+Source composition:
```

Comment:
Are these supposed to be in "classic" BIOM format? The sample ID for both of these is "Relative Abundance" which is confusing.